common/src/main/java/io/netty/util/internal/svm/PlatformDependent0Substitution.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 The Netty Project
+ * Copyright 2020 The Netty Project
```

Comment:
please dont adjust copyright years (same is true for other files that are not new).

Reply:
Meeeeh… Sorry, my bad. That was me being clever with find / sed.